Jellyfin.Plugin.Anime/Providers/AniDB/Identity/AniDbTitleDownloader.cs

```diff
@@ -18,7 +18,8 @@ public class AniDbTitleDownloader : IAniDbTitleDownloader
         /// <summary>
         /// The URL for retrieving a list of all anime titles and their AniDB IDs.
         /// </summary>
-        private const string TitlesUrl = "http://anidb.net/api/animetitles.xml.gz";
+        private const string TitlesUrl = "http://anidb.net/api/anime-titles.xml.gz";
```

Comment:
The old url works, but the hyphen is in the [official docs](https://wiki.anidb.net/API#Anime_Titles)